semgrep-core/tests/js/misc_destructuring_2.sgrep

```diff
@@ -0,0 +1,3 @@
+function $F ({$FOO}, ...) {
```

Comment:
I would also add the simpler  const $X = foo.$X; more obvious testcase.

Reply:
Oddly enough you can not write this pattern 🤷

Reply:
Apparently you can only write this pattern via transpilation from destructuring; it can not be parsed directly.